src/scripts/main.js

```diff
@@ -18,10 +18,28 @@ const food = {
   },
 };
 
+const body = document.querySelector('body');
+const div = document.createElement('div');
+
+div.setAttribute('id', 'tree');
+body.prepend(div);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    createTree(li, data[key]);
```

Comment:
The `createTree` function recursively calls itself for every key in the `data` object. However, this can lead to the creation of unnecessary empty `<ul>` elements for keys that have empty objects as values (like `Wine`, `Schnaps`, `Cherry`, etc.). Consider adding a condition to check if `data[key]` is not an empty object before calling `createTree` recursively.